README.md

```diff
@@ -29,10 +37,45 @@ npm test
 ## Implementation
 
 ### VC-API
-To add your implementation to this test suite see the [README here](https://github.com/w3c-ccg/vc-api-test-suite-implementations).
-Add the tag `vc-api` to the issuers and verifiers you want tested. To run the tests, some implementations require client secrets
+To add your implementation to this test suite You will need to add 3 endpoints to your implementation manifest.
+- A credentials issuer endpoint (/credentials/issue) in the `issuers` property.
+- A credentials verifier endpoint (/credentials/verify) in the `verifiers` property.
+- A presentations verifier (presentations/verify) in a new property `vpVerifiers`.
```

Comment:
We need to show example requests and responses that the test suite will send to these endpoints so developers know what to implement. Even though it's documented in the vc-api, having all docs on how to integrate here will be helpful.

Reply:
Addressed here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/17/commits/faf512cd9fb90ae0e2d8745aedbbfe247f291166

and in PR from @msporny so marking as resolved